src/sinopiaSearch.js

```diff
@@ -70,9 +70,17 @@ export const getTemplateSearchResults = async (query) => {
   }
 
   const url = `${Config.searchHost}${Config.templateSearchPath}`
-
   return fetch(url, { method: 'POST', headers: { 'Content-Type': 'application/json' }, body: JSON.stringify(body) })
-    .then(resp => resp.json())
+    .then((resp) => {
+      if (resp.status > 300) {
```

Comment:
Can you say more about what the significance of 300 is here? If Trellis does return a 300, do you not want this to catch? I might have expected `>= 300` or `> 299` instead to flag all non-3xx, 4xx, and 5xx responses.

Reply:
Thanks for catching this! No particular significance of using 300 code for handling, changing to use `>=` operators.

Reply:
@jermnelson super! thanks :smile: